### PR TITLE
Setting evita script to colour mindmup nodes.

### PR DESCRIPTION
**Pull request by bevan-watkiss**

### CODE REVIEW

File: scripts/add-evita.py [hunk 1]
```diff
@@ -42,7 +42,11 @@ def parse_evita_raps(node):
 		    node.update({'attr': dict()})
 
 		attr = node.get('attr')
-
+		
+		if len(evita_line) != 10:
+			print ("EVITA:: tag should have exactly 9 elements in attack vector node", node,"where it only has",len(evita_line)-1)
```
Comment (BenGardiner) [inline scripts/add-evita.py]: nice check, thanks.

can you please use the `%` operator for consistency in the repo? e.g. "EVITA:: tag should ... vector node %s where it " % (node, ...)

File: scripts/add-evita.py [hunk 2]
```diff
@@ -165,17 +169,26 @@ def derive_evita_apt(node):
...
 	else:
 		apt = 1
+		apt_colour = "#00FFFF"
 	#TODO support non-zero controllability
 
 	attrs.update({'evita_apt': apt})
+	if attrs.get('style', None) is None:
+	    attrs.update({'style': dict()})
+	style = attrs.get('style')
+	style.update({"background":apt_colour})
```
Comment (BenGardiner) [inline scripts/add-evita.py]: for consistency in the repo:
* single-quote strings please
* space after `:` please

File: scripts/mindmup_as_attack_trees.py [hunk 3]
```diff
@@ -423,7 +423,25 @@ def update_node_apt(node, apt):
 	if node.get('attr', None) is None:
 		node.update({'attr': dict()})
 
-	node.get('attr').update({'evita_apt': apt})
```
Comment (BenGardiner) [inline scripts/mindmup_as_attack_trees.py]: oh, in multiple places...

the .py files are now setup to import common functions from this .py file. c.f. ` from mindmup_as_attack_trees import *` in the `add-evita.py`.

So you should define this color LUT as a function in this module and call it from the callsite of `.update()` referencing `apt_colour` below and also from the analog callsite in the `add-evita.py` script

**Review by BenGardiner — CHANGES_REQUESTED** [review]
thanks, nice improvement for when you're iterating in the mindmup UI

File: scripts/add-evita.py (hunk 1 above)
Comment (bevan-watkiss) [reply scripts/add-evita.py]: Forgot that was there.  Bonus fix.  Done

File: scripts/add-evita.py (hunk 2 above)
Comment (bevan-watkiss) [reply scripts/add-evita.py]: Replaced with call to general function

File: scripts/mindmup_as_attack_trees.py (hunk 3 above)
Comment (bevan-watkiss) [reply scripts/mindmup_as_attack_trees.py]: Made into a general update function.  So only one place.

**Review by BenGardiner — APPROVED** [review]
nice

**Pull request merged**